metadata-ingestion/src/datahub/ingestion/source/ge_data_profiler.py

```diff
@@ -697,6 +716,9 @@ def generate_dataset_profile(  # noqa: C901 (complexity)
                             1, unique_count / non_null_count
                         )
 
+            if not profile.rowCount:
```

Comment:
This causes the mysql golden file changes

Reply:
yup those look reasonable